chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md

```diff
@@ -447,11 +447,11 @@ $$
 \begin{aligned}
 f(0,0) & = a, \\
 \nabla f (0,0) & = \begin{bmatrix}b_1 \\ b_2\end{bmatrix}, \\
-\mathbf{H} f (0,0) & = \begin{bmatrix}2 c_{11} & c_{12} \\ c_{12} & 2c_{22}\end{bmatrix}.
+\mathbf{H} f (0,0) & = \begin{bmatrix}2 c_{11} & c_{12} \\ c_{12} & 2c_{22}\end{bmatrix},
```

Comment:
Here is the end of formula, so we use period.

Reply:
Can you change back? Thanks!

Reply:
The sentence of "If we look..." before these formulas is not finish, so I changed period and modify the sentence below which miss a verb to form a complete sentence.

Reply:
I have changed it in ca9e07575b26e25e4d48b79b99ed55837fe307f6.